src/app/screens/homepage/index.ts

```diff
@@ -182,5 +184,6 @@ export class HomePage {
     this.triggerStartButton()
     this.triggerSettingsButton()
     new InfoContentProvider()
+    new GameSettingsView(this.app, new Audio())
```

Comment:
Create in this folder this.audio = new Audio() and then pass it to settingsView. Audio is Application feature not only SettingsView